firmware/robot2015/CMakeLists.txt

```diff
@@ -1 +1,7 @@
-add_subdirectory(cpu)
+# we'll go ahead and set the AVR toolchain cmake file too for consisteny in defining all toolchain setup files
+set(ATTINY_TOOLCHAIN_FILE ${CMAKE_CURRENT_LIST_DIR}/attiny_toolchain.cmake)
+set(ATTINY_TOOLCHAIN_FILE ${ATTINY_TOOLCHAIN_FILE} PARENT_SCOPE)
```

Comment:
ok, and I think we might have to do something weird to compile for the attiny441. I haven't looked into at all really, but gcc doesn't know about it when you pass it as a compiler flag.